src/api/ledger/parse/utils.js

```diff
@@ -4,6 +4,15 @@ const _ = require('lodash');
 const transactionParser = require('ripple-lib-transactionparser');
 const toTimestamp = require('../../../core/utils').toTimestamp;
 const utils = require('../utils');
+const BigNumber = require('bignumber.js');
+
+function adjustQualityForXRP(quality: string, takerGetsCurrency: string,
+    takerPaysCurrency: string) {
+  const shift = (takerGetsCurrency === 'XRP' ? 6 : 0)
```

Comment:
It's not so much the terms as much as their semantics. I'll try and
articulate it better, but on something else atm. I did look at bitstamp api
the other day, and noticed it returned asks/bids with a type buy/sell.

Anyway, as you said, you'll be seeking feedback from people after this
initial phase is complete, so I'm happy to take it on faith for the moment
and reserve judgement.